src/Symfony/Component/Form/Tests/Extension/Validator/DataTransformer/ViewObjectTransformer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Symfony\Component\Form\Tests\Extension\Validator\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+* Transforms Object to array for view
+*
+* @author Charles J. C. Elling <charles@denumeris.com>
+*/
+class ViewObjectTransformer implements DataTransformerInterface {
```

Comment:
I was wrong, it was posible to make the changes in the _Form_ class but the inclusion of the new method  to the _FormInterface_ will have to wait.